docs/modules/ROOT/pages/team-and-users.adoc

```diff
@@ -1,23 +1,85 @@
-= Team and Users
+= Managing Teams and Users
+:description: By default, accounts include a single owner. If you want to collaborate with other team members, you can allow them to access specific clusters by inviting them to a team and assigning them roles based on what actions they need to perform on the team's clusters.
 
-You can create teams and invite users to these teams with different roles. By default, your account is single that means you do not own or belong to a team. To create a team, click on *Account* > *Team & Users* as below:
+{description}
 
-image:team-and-users.png[Choose an existing team or create a new one]
+== Team Roles
 
-You need to give a name then click on *Create Team* button.
-Once, your team is created, you can invite users to the team as below:
+In each team, users can have one of the following roles:
 
-image:user-roles.png[A dropdown menu with a list of user roles]
+[cols="2a,1a,1a,1a,1a"]
+|===
+||Admin|Developer|Finance|View only
 
-There are 4 different roles that you can assign to the team members:
+|Creating new clusters for the team
+|yes
+|yes
+|no
+|no
 
-- View Only: This role can only view the charts and metrics but cannot create, stop or update the clusters.
-- Developer: Thıs role can update the cluster in addition to seeing the charts and metrics. But it cannot update and access payment settings and team settings.
-- Admin: This role has access to everything including payment and team settings.
-- Finance: This role has only access to payment setting and billing information.
+|Managing billing settings
+|yes
+|no
+|yes
+|no
 
-After creating your team, refresh your dashboard. You will see a dropdown menu selection on top-right corner of your dashboard:
+|Redeeming credits
+|yes
+|no
+|yes
+|no
+
+|Managing API keys
+|yes
+|yes
+|no
+|no
+
+|Updating cluster settings
+|yes
+|yes
+|no
+|no
+
+|Viewing charts and metrics
+|yes
+|yes
+|yes
+|yes
+
+
+|===
+
+== Creating a Team
+
+. Go to *Settings* > *Team & Users*.
++
+image:team-and-users.png[Create a new team]
+
+. Enter a team name and click *Create Team*.
+
+Now that you've created a team, you can invite users to it and assign them roles.
+
+== Inviting Users to a Team
+
+If you've created one or more teams, you can invite users to them and assign those users one of <<user-roles, four roles>>.
+
+. Select an existing team from the dropdown.
++
+image:choose-team.png[Choose an existing team]
+
+. Enter the email address of the team member that you want to add and assign them a <<user-roles, role>> from the dropdown.
++
+image:user-roles.png[A dropdown menu with a list of roles]
```

Comment:
Image is not found
![Screen Shot 2021-09-02 at 15 20 51](https://user-images.githubusercontent.com/1237982/131842547-69349d14-30c2-47f6-a9a5-93fa71541b7b.png)